lib/calabash/http/retriable_client.rb

```diff
@@ -25,7 +25,7 @@ def get(request, options={})
             @client.get(@server.endpoint + request.route, request.params)
           end
         rescue => e
-          raise HTTP::Error.new(e.message)
+          raise Calabash::HTTP::Error.new(e.message)
```

Comment:
@jmoody I guess @TobiasRoikjer is right?

`::HTTP::Error` != `::Calabash::HTTP::Error`, but if you're somewhere inside the `Calabash` namespace, `HTTP::Error` will resolve to `Calabash::HTTP::Error`.

Except, of course, if you haven't defined `Calabash::HTTP::Error`. Which may be the case for your rspec test?